docs/pages_en/reference/cheat_sheet.md

```diff
@@ -22,7 +22,7 @@ werf converge --repo ghcr.io/group/project --use-custom-tag "%image%-$CI_JOB_ID"
 
 You can create a pipeline tailored to your needs using the commands below.
 
-Running most of the commands will first cause the missing images to be rebuilt. You can skip the rebuild by using the `--skip-build` flag. Make sure that the necessary images are built beforehand using the `werf build` command.
+Running most of the commands will first check for images in the local cache or the specified repo. Build instructions will run for missing images. For some scenarios, e.g. running tests in CI system, it is more convenient to use `werf build` to build images first and the use them in other commands with flag `--require-built-images`. The command will exit with error in case of missing images.
```

Comment:
```suggestion
Running most commands will first check for images in the specified repo. Build instructions will run for missing images. For some scenarios, e.g. running tests in CI system, it is more convenient to use `werf build` to build images first and then strictly use the same images on other steps with flag `--require-built-images`. The step command will exit with an error in case of missing images.
```